### ISSUE TITLE
Feature: Modify alter table column parser. 

### ISSUE THREAD

**Issue by TCeason** [datafuselabs/databend#13790]
**Summary**

ADD COLUMN -> ADD [ COLUMN ]

Now in Databend:

https://databend.rs/sql/sql-commands/ddl/table/alter-table-column

```sql
-- Add a column to the end of the table
ALTER TABLE [IF EXISTS] [database.]<table_name> 
ADD COLUMN <column_name> <data_type> [NOT NULL | NULL] [DEFAULT <constant_value>];

-- Add a column to a specified position
ALTER TABLE [IF EXISTS] [database.]<table_name> 
ADD COLUMN <column_name> <data_type> [NOT NULL | NULL] [DEFAULT <constant_value>] [FIRST | AFTER <column_name>]

-- Add a virtual computed column
ALTER TABLE [IF EXISTS] [database.]<table_name> 
ADD COLUMN <column_name> <data_type> AS (<expr>) VIRTUAL;

-- Convert a stored computed column to a regular column
ALTER TABLE [IF EXISTS] [database.]<table_name> 
MODIFY COLUMN <column_name> DROP STORED;

-- Rename a column
ALTER TABLE [IF EXISTS] [database.]<table_name>
RENAME COLUMN <column_name> TO <new_column_name>;

-- Change the data type of one or multiple columns
ALTER TABLE [IF EXISTS] [database.]<table_name> 
MODIFY COLUMN <column_name> <new_data_type> [DEFAULT <constant_value>][, COLUMN <column_name> <new_data_type> [DEFAULT <constant_value>], ...]

-- Set / Unset masking policy for a column
ALTER TABLE [IF EXISTS] [database.]<table_name>
MODIFY COLUMN <column_name> SET MASKING POLICY <policy_name>

ALTER TABLE [IF EXISTS] [database.]<table_name>
MODIFY COLUMN <column_name> UNSET MASKING POLICY

-- Remove a column
ALTER TABLE [IF EXISTS] [database.]<table_name> 
DROP COLUMN <column_name>;
```

Can modify to this:

```sql
-- Add a column to the end of the table
ALTER TABLE [IF EXISTS] [database.]<table_name> 
ADD [ COLUMN ] <column_name> <data_type> [NOT NULL | NULL] [DEFAULT <constant_value>];

-- Add a column to a specified position
ALTER TABLE [IF EXISTS] [database.]<table_name> 
ADD [ COLUMN ] <column_name> <data_type> [NOT NULL | NULL] [DEFAULT <constant_value>] [FIRST | AFTER <column_name>]

-- Add a virtual computed column
ALTER TABLE [IF EXISTS] [database.]<table_name> 
ADD [ COLUMN ] <column_name> <data_type> AS (<expr>) VIRTUAL;

-- Convert a stored computed column to a regular column
ALTER TABLE [IF EXISTS] [database.]<table_name> 
MODIFY [ COLUMN ] <column_name> DROP STORED;

-- Rename a column
ALTER TABLE [IF EXISTS] [database.]<table_name>
RENAME [ COLUMN ] <column_name> TO <new_column_name>;

-- Change the data type of one or multiple columns
ALTER TABLE [IF EXISTS] [database.]<table_name> 
MODIFY [ COLUMN ] <column_name> <new_data_type> [DEFAULT <constant_value>][, COLUMN <column_name> <new_data_type> [DEFAULT <constant_value>], ...]

-- Set / Unset masking policy for a column
ALTER TABLE [IF EXISTS] [database.]<table_name>
MODIFY [ COLUMN ] <column_name> SET MASKING POLICY <policy_name>

ALTER TABLE [IF EXISTS] [database.]<table_name>
MODIFY [ COLUMN ] <column_name> UNSET MASKING POLICY

-- Remove a column
ALTER TABLE [IF EXISTS] [database.]<table_name> 
DROP [ COLUMN ] <column_name>;
```

**Comment by TCeason** [comment]
If this fixed, dbeaver alter table t add column_name can be worked? cc @hantmac

**Comment by sundy-li** [comment]
fixed in #14533
